libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ReverificationStatus.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@Getter
+@ExcludeFromGeneratedCoverageReport
+public enum ReverificationStatus {
+    SUCCESS("success"),
+    FAILED("failed");
+
+    private final String status;
```

Comment:
Is this status string used anywhere? If not simpler to remove and just have:
```
public enum ReverificationStatus {
    SUCCESS,  FAILED;
}
```

Reply:
TBH it could also just be a boolean (although an enum does give us more flexibility to extend in future)